pyomo/core/tests/unit/test_compare.py

```diff
@@ -28,7 +28,7 @@ class TestConvertToPrefixNotation(unittest.TestCase):
     def test_linear_expression(self):
         m = pe.ConcreteModel()
         m.x = pe.Var([1, 2, 3, 4])
-        e = LinearExpression(constant=3, linear_coefs=m.x.keys(), linear_vars=m.x.values())
+        e = LinearExpression(constant=3, linear_coefs=m.x.keys(), linear_vars=list(m.x.values()))
```

Comment:
I am surprised that you don't also need `linear_coefs=list(m.x.keys())`

Reply:
Good point. I'll add it.